docs/sql/index.md

```diff
@@ -91,3 +93,56 @@ POST _opendistro/_sql/_explain
   "size": 50
 }
 ```
+
+### Syntax analysis exception
+
+
+If you get the following "Invalid SQL query" error:
+
+
+```json
+{
+  "reason": "Invalid SQL query",
+  "details": "Failed to parse query due to offending symbol [:] at: 'SELECT * FROM xxx WHERE xxx:' <--- HERE...
+    More details: Expecting tokens in {<EOF>, 'AND', 'BETWEEN', 'GROUP', 'HAVING', 'IN', 'IS', 'LIKE', 'LIMIT',
+    'NOT', 'OR', 'ORDER', 'REGEXP', '*', '/', '%', '+', '-', 'DIV', 'MOD', '=', '>', '<', '!',
+    '|', '&', '^', '.', DOT_ID}",
+  "type": "SyntaxAnalysisException"
+}
+```
+
+To resolve this error:
+
+1. Check if your syntax follows the MySQL grammar.
+1. If your syntax is correct, disable strict query analysis:
```

Comment:
You can make this entire section a single list. So:

1. Check
1. If

   ```json
   code block here
   ```

1. Run the query again

Reply:
Done.